docs/api.rst

```diff
@@ -818,6 +818,27 @@ Integrations
     :param payload: The raw event payload data.
     :type payload: :class:`RawIntegrationDeleteEvent`
 
+Audit Logs
+++++++++++
+
+.. function:: on_audit_log_entry_create(entry)
```

Comment:
This is now missing from the new `Event` enum :sweat_smile:

Reply:
f46e769e120e62f9d4f16cdf28e8ea2639e0c6d1